sentry_ldap_auth/backend.py

```diff
@@ -82,3 +74,13 @@ def get_or_create_user(self, username, ldap_user):
             )
 
         return model
+
+    def _add_user_in_organization(self, organizations, user, member_role, has_global_access):
```

Comment:
nit.. rename `organizations` to `org`
```suggestion
    def _add_user_in_organization(self, org, user, member_role, has_global_access):
```